modules/xmpp/JingleSessionPC.js

```diff
@@ -22,6 +22,148 @@ const logger = getLogger(__filename);
 const IQ_TIMEOUT = 10000;
 
 /**
+ * The task which is executed on {@link JingleSessionPC.modificationQueue}
+ * which synchronizes modifications to the underlying
+ * {@link TraceablePeerConnection} state. All work that could change how either
+ * local or remote description will look should be executed through that queue.
+ *
+ * A task instance should be queued with {@link JingleSessionPC._doRenegotiate}
+ * which wraps common checks around renegotiation process (sRD/sLD cycle) and
+ * makes sure to send 'source-add/'source-remove' notifications in order to
+ * broadcast local stream description to all conference participants
+ * (through Jicofo).
+ *
+ * Class takes as an argument a worker function which should modify the
+ * instance state in a way which would reflect the result of the job performed.
```

Comment:
could you clarify what instance is being referred to here (and in the following sentence)?